examples/slurm/submit_multicpu.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -l
+
+#SBATCH --job-name=multicpu
+#SBATCH --nodes=2                       # number of Nodes
+#SBATCH --ntasks-per-node=1             # number of MP tasks
+#SBATCH --exclusive
+#SBATCH --output=output/torch_%A.log
+#SBATCH --cpus-per-task=56
```

Comment:
What happens if we don't pass this? I'm just thinking that if it doesn't make much of a difference then let's not pass it, since someone else's machine can have a different number of cores/threads